packages/node_modules/@ciscospark/plugin-memberships/src/memberships.js

```diff
@@ -24,6 +39,177 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
  * @name Memberships
  */
 const Memberships = SparkPlugin.extend({
+  /**
+   * Connect to the web socket to listen to incoming messages.
+   * @returns {Promise}
+   */
+  listen() {
+    // Create a common envelope that we will wrap all events in
+    this.eventEnvelope = {
+      resource: SDK_EVENT_RESOURCE.MEMBERSHIPS,
+      // id -- webhook id concept does not correlate to SDK socket event
+      // name -- webhook name concept does not correlate to SDK socket event
+      // targetUrl -- targetUrl concept does not correlate to SDK socket event
+      // secret -- secret concept does not correlate to SDK socket event
+      ownedBy: SDK_EVENT_OWNER.CREATOR,
+      status: SDK_EVENT_STATUS.ACTIVE,
+      created: new Date().toISOString(),
+      data: {}
+    };
+    const that = this;
```

Comment:
You don't need to do this with arrow functions. See the section ["Functionality: lexical scoping “this”"](https://ultimatecourses.com/blog/es6-arrow-functions-syntaxes-and-lexical-scoping).

Reply:
Removed